popup/popup.css

```
@@ -30,41 +44,25 @@ h3 {
 
 }
 
-
-
 button {
   width: 100%;
-  background: none;
-  /*color: white;*/
-  border: 2px solid #2980b9;
-  padding: 5px;
-  /*font-size: 1.5em;*/
-  margin: 10px 0 0 0;
+  background-color: rgba(22, 23, 3, 0.6);
+  padding: 10px;
+  margin: 8px 0;
   cursor: pointer;
-  transition: 0.5s;
+  box-shadow: 0 6px #141414;
+  transition: 0.1s;
+  border: none
 }
 button:disabled {
+  background-color: rgb(31 41 55);
+  color: rgb(62 82 110);
+  box-shadow: 0 6px #141414;
   cursor: default;
 }
-button:hover/*, button:focus*/ {
-  /*width:80%;*/
-  border-color:#5d94b9;
-  /*transform: scale(1.05);*/
-}
-
-.cg {
-  border-color: #464d56;
-  background-color: #464d56;
-}
-.cg:hover {
-  border-color: #41464a;
-  background-color: #41464a;
-}
-.cg-yellow {
-  color: #f2bb13;
-}
-.cg-blue {
-  color: #1a99aa;
+button:active {
```

Il y aurait pas un autre pseudo-element ? Sur mon pad je n'ai pas besoin de clicker (juste toucher) et le bouton ne s'enfonce pas. En revanche si je click là il s'enfonce bien.

---

C'était le comportement voulu, dans la vrai vie quand tu survoles une touche elle ne s'enfonce pas mais quand tu appuis dessus si. Je dis ça mais on trouve de tout sur internet, tu pourras surement trouver l'inverse facilement.

On pourrait ajouter un hover si tu veux un effet au survol je pense que le faire en deux temps en enfoncant à moitié puis totalement le bouton pourrait être sympa, je regarde demain en live avec seb, je resolve parce que pas "critique"

---

Il y a bien une histoire de délai avec les touchpad. Rajouter `transition: none` contourne le problème de délai. Le seul défaut est que le bouton s'enfonce instantanément... voir si c'est pas trop grave. Merci stackoverflow : https://stackoverflow.com/a/63942277
```suggestion
button:active {
  /* for touchpad */
  transition: none;
```

---

Commit ça et on merge ?